packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -66,8 +67,17 @@ import {
 // eslint-disable-next-line no-process-env
 const isCI = process.env.CI
 
+/**
+ * @deprecated - No longer used
+ */
 export interface InitOptions {
```

Comment:
This Interface is never used. Can I remove?

Reply:
Do you want to remove it? Maybe better to do it in a separate PR